testsuite/run_sets/reposync.yml

```diff
@@ -13,7 +13,6 @@
 - features/reposync/srv_sync_products.feature
 - features/reposync/srv_enable_sync_products.feature
 - features/reposync/srv_wait_for_reposync.feature
-- features/reposync/srv_check_reposync.feature
```

Comment:
Realised this one is the correct one and the one in init_clients should not be there, currently fixing.